.phpcs.xml

```diff
@@ -21,18 +21,17 @@
   <exclude-pattern>./test/helpers/StoreAndHeadersFactory.php</exclude-pattern>
   <exclude-pattern>./test/integration/SSIWovnIndexSampleTest.php</exclude-pattern>
   <exclude-pattern>./test/integration/WovnIndexSampleTest.php</exclude-pattern>
-  <exclude-pattern>./test/unit/APITest.php</exclude-pattern>
   <exclude-pattern>./test/unit/HeadersTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/html/HtmlConverterTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/html/HtmlReplaceMakerTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/LangTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/modified_vendor/SimpleHtmlDomTest.php</exclude-pattern>
-  <exclude-pattern>./test/unit/SSITest.php</exclude-pattern>
   <exclude-pattern>./test/unit/StoreTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/UrlTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/utils/request_handlers/CurlRequestHandlerTest.php</exclude-pattern>
   <exclude-pattern>./test/unit/utils/request_handlers/FileGetContentsRequestHandlerTest.php</exclude-pattern>
-  <exclude-pattern>./test/unit/UtilsTest.php</exclude-pattern>
+  <!-- FIXME: needs indent fixed to solve last errors -->
+  <exclude-pattern>./test/unit/APITest.php</exclude-pattern>
```

Comment:
Can it be included in this PR?

Reply:
No, if i fix indent it makes PR difficult to review because of every line changing. So I deactivated the indent rule until other errors are fixed. But in the mean time, some PSR2 rules seem to be too dependent to the ident being 4 spaces, thus creating errors that should not exist.

Thanks for asking (although I explained already ;) )

Reply:
<img width="409" alt="screen shot 2018-10-29 at 11 10 41 am" src="https://user-images.githubusercontent.com/2074765/47625894-54790100-db6b-11e8-83fd-c7a11feccff2.png">

Turning on `No whitespace` would help for reviewing just indent changes